src/test/java/gift/MemberRepositoryTest.java

```diff
@@ -0,0 +1,40 @@
+package gift;
+
+import gift.entity.Member;
+import gift.repository.MemberRepository;
+import jakarta.transaction.Transactional;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class MemberRepositoryTest {
+
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @Test
+    void save() {
+        Member member = new Member("test@example.com", "pw123", "USER");
+        Member saved = memberRepository.save(member);
+        assertThat(saved.getId()).isNotNull();
+        assertThat(saved.getEmail()).isEqualTo("test@example.com");
+        assertThat(saved.getRole()).isEqualTo("USER");
```

Comment:
```suggestion
assertAll(
            () -> assertThat(saved.getId()).isNotNull(),
            () -> assertThat(saved.getEmail()).isEqualTo("test@example.com"),
            () -> assertThat(saved.getRole()).isEqualTo("USER")
        );
```
이렇게 사용한다면 어떤 장단점이 있을까요?